todos/client.coffee

```diff
@@ -56,12 +56,13 @@ $ ->
     newTodo.val ''
     # Insert the new todo before the first completed item
     items = listModel.get()
-    for todo, i in items
-      break if todo?.completed
-    listModel.insert i,
-      id: model.id()
+    if items
+      for todo, i in items
+        break if todo?.completed
+    newTodoId = model.add 'todos',
```

Comment:
Avoid implicit creation errors by calling `model.add` first, and then `listModel.insert`